api/rootcerts.go

```diff
@@ -0,0 +1,126 @@
+package api
```

Comment:
Since this file is very similar to https://github.com/hashicorp/go-rootcerts/blob/master/rootcerts.go, which is under the MPL, I believe we need a copyright header like:

```
// Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0
//
// See https://github.com/hashicorp/go-rootcerts's rootcerts.go as of bb0b55efd5a1de16cb10686096b9596cfbab1fad.
```

to reflect the dual-source nature of this new file.